### PR TITLE
config: create `BootstrapPeers` p2p config parameter

### PR DESCRIPTION
**Pull request by cmwaters**
Closes: #6363

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -551,8 +551,16 @@ type P2PConfig struct { //nolint: maligned
...
 	// We only use these if we can’t connect to peers in the addrbook
+	// NOTE: not used by the new PEX reactor. Please use BootstrapPeers instead.
+	// TODO: Remove once p2p refactor is complete
+	// ref: https://github.com/tendermint/tendermint/issues/5670
 	Seeds string `mapstructure:"seeds"`
 
+	// Comma separated list of peers to be added to the peer store
+	// on startup. Either BootstrapPeers or PersistentPeers are
+	// needed for peer discovery
+	BootstrapPeers string `mapstructure:"initial-peers"`
```
Comment (alexanderbez) [inline config/config.go:562]: There is a conflict between the field name and the key name.

Comment (alexanderbez) [inline config/config.go:562]:
```suggestion
	BootstrapPeers string `mapstructure:"bootstrap-peers"`
```

**Review by alexanderbez — APPROVED** [review]
Nice! This can't be hooked up until the seed work/pex v2 is complete, right?

File: config/config.go (hunk 1 above)
Comment (cmwaters) [reply config/config.go:562]: nice spotting. I initially called this initial-peers but then changed it

File: node/node.go [hunk 2]
```diff
@@ -629,6 +629,14 @@ func createPeerManager(
 		options.PersistentPeers = append(options.PersistentPeers, address.NodeID)
 	}
 
+	for _, p := range splitAndTrimEmpty(config.P2P.BootstrapPeers, ",", " ") {
+		address, err := p2p.ParseNodeAddress(p)
+		if err != nil {
+			return nil, fmt.Errorf("invalid peer address %q: %w", p, err)
+		}
+		peers = append(peers, address)
```
Comment (cmwaters) [inline node/node.go:637]: It gets added to the peer manager here before we even start the pex reactor @alexanderbez

Comment (alexanderbez) [reply node/node.go:637]: Yup, this looks correct!

**Pull request merged**